assets/js/compare-commits.js

```diff
@@ -2,41 +2,66 @@
-      $row.nextAll().find("[name='compare-to']").prop('disabled', true)
-      $row.find("[name='compare-to']").prop('disabled', true)
+    if (radioInput.getAttribute('name') === 'compare-from') {
+      prevAll(row).forEach(
+        r => Array.from(r.querySelectorAll('[name="compare-to"]')).forEach(item => item.removeAttribute('disabled'))
+      )
+      nextAll(row).forEach(
+        r => Array.from(r.querySelectorAll('[name="compare-to"]')).forEach(item => item.setAttribute('disabled', true))
+      )
+      Array.from(row.querySelectorAll('[name="compare-to"]')).forEach(item => item.setAttribute('disabled', true))
     } else {
-      $row.prevAll().find("[name='compare-from']").prop('disabled', true)
-      $row.nextAll().find("[name='compare-from']").prop('disabled', false)
-      $row.find("[name='compare-from']").prop('disabled', true)
+      prevAll(row).forEach(
+        r => Array.from(r.querySelectorAll('[name="compare-from"]')).forEach(item => item.setAttribute('disabled', true))
+      )
+      nextAll(row).forEach(
+        r => Array.from(r.querySelectorAll('[name="compare-from"]')).forEach(item => item.removeAttribute('disabled'))
+      )
+      Array.from(row.querySelectorAll('[name="compare-from"]')).forEach(item => item.setAttribute('disabled', true))
     }
   }
 
-  $(".commits-list input[name^='compare']").on('change', function() {
-    toogleRadioInput($(this))
-  })
+  window.addEventListener('DOMContentLoaded', () => {
+    Array.from(document.querySelectorAll('.commits-list input[name^="compare"]:checked')).forEach(radioElement => toogleRadioInput(radioElement))
 
-  $(document).ready(function() {
-    $(".commits-list input[name^='compare']:checked").each(function() {
-      toogleRadioInput($(this))
+    Array.from(document.querySelectorAll('.commits-list input[name^="compare"]')).forEach((el) => {
+      el.addEventListener('change', function() {
+        toogleRadioInput(this)
+      })
     })
-  })
 
-  $('.commits-compare-form').on('submit', function() {
-    var $form = $(this)
-    var $fromInput = $form.find("input[name='from']")
-    var $toInput = $form.find("input[name='to']")
-    var compareFrom = $(".commits-list input[name='compare-from']:checked").val()
-    var compareTo = $(".commits-list input[name='compare-to']:checked").val()
+    Array.from(document.querySelectorAll('.commits-compare-form')).forEach((el) => {
+      el.addEventListener('submit', function(e) {
+        const fromInput = this.querySelector("input[name='from']")
+        const toInput = this.querySelector("input[name='to']")
+        const compareFrom = document.querySelector(".commits-list input[name='compare-from']:checked").value
+        const compareTo = document.querySelector(".commits-list input[name='compare-to']:checked").value
 
-    $fromInput.val(compareFrom)
-    $toInput.val(compareTo)
+        fromInput.value = compareFrom
+        toInput.value = compareTo
```

Comment:
Les valeurs de `fromInput` et `toInput` pourraient être utilisées directement sans variable, étant donné que le nom de la variable n'apporte pas grand chose et qu'elles ne sont utilisées qu'une fois.

```suggestion
        this.querySelector("input[name='from']").value = compareFrom
        this.querySelector("input[name='to']").value = compareTo
```

Reply:
ping @firm1 ?